From 75c03c5d2a00d255996787ddb2818608dbe8e6dd Mon Sep 17 00:00:00 2001
From: Bo Yang <bo.yang@amlogic.com>
Date: Mon, 10 Aug 2015 01:05:54 -0700
Subject: [PATCH 5758/5965] Revert "PD#108095: odroidc: fix compilation"

This commit is a patch for the commit:
PD#108095: odroidc: merge from odroidc and fix crashes

This reverts commit 600b484ebc12116dccaca40bb8024fcc9b0163f6.

Change-Id: Ib203bea5c267343c585e2a8abaaf5e793b011029
---
 arch/arm/mach-meson8b/include/mach/sd.h | 2 --
 drivers/amlogic/mmc/amlsd_of.c          | 2 --
 2 files changed, 4 deletions(-)

diff --git a/arch/arm/mach-meson8b/include/mach/sd.h b/arch/arm/mach-meson8b/include/mach/sd.h
index efc70d897c01..dca8683c5712 100644
--- a/arch/arm/mach-meson8b/include/mach/sd.h
+++ b/arch/arm/mach-meson8b/include/mach/sd.h
@@ -94,9 +94,7 @@ struct amlsd_platform {
     unsigned int gpio_dat3;
     unsigned int jtag_pin;
 
-#if defined(CONFIG_MACH_MESON8B_ODROIDC)
 	unsigned int gpio_volsw;
-#endif
     int is_sduart;
     bool is_in;
     bool is_tuned;                      /* if card has been tuning */
diff --git a/drivers/amlogic/mmc/amlsd_of.c b/drivers/amlogic/mmc/amlsd_of.c
index 0146d101e0cd..37290bfef9d6 100755
--- a/drivers/amlogic/mmc/amlsd_of.c
+++ b/drivers/amlogic/mmc/amlsd_of.c
@@ -223,9 +223,7 @@ int amlsd_get_platform_data(struct platform_device* pdev,
         SD_PARSE_GPIO_NUM_PROP(child, "jtag_pin", str, pdata->jtag_pin);
 		SD_PARSE_U32_PROP(child, "card_type", prop, pdata->card_type);
         SD_PARSE_GPIO_NUM_PROP(child, "gpio_dat3", str, pdata->gpio_dat3);
-#if defined(CONFIG_MACH_MESON8B_ODROIDC)
         SD_PARSE_GPIO_NUM_PROP(child, "gpio_volsw", str, pdata->gpio_volsw);
-#endif
 
 		pdata->port_init = of_amlsd_init;
 		pdata->pwr_pre = of_amlsd_pwr_prepare;
-- 
2.19.0

